public/app/plugins/datasource/elasticsearch/elastic_response.ts

```diff
@@ -391,72 +391,103 @@ export class ElasticResponse {
   }
 
   getTimeSeries() {
-    const seriesList = [];
+    if (
+      this.targets.some((target: any) => target.metrics.some((metric: any) => metric.settings?.useNewFormat === true))
```

Comment:
this is the only line that changed in `public/app/plugins/datasource/elasticsearch/elastic_response.ts` compared to https://github.com/grafana/grafana/pull/26233